pyo3-macros-backend/src/method.rs

```diff
@@ -196,10 +196,11 @@ impl SelfType {
                 holders.push(quote_spanned! { *span =>
                     #[allow(clippy::let_unit_value)]
                     let mut #holder = _pyo3::impl_::extract_argument::FunctionArgumentHolder::INIT;
+                    let mut #slf = _pyo3::Borrowed::from_ptr(#py, #slf);
```

Comment:
Would `BoundRef` work here? Just to be a extra cautious about the `Borrowed` lifetime.

Reply:
Thanks, yes using `BoundRef` instead of `Borrowed` is a much safer idea for macro code. I think I need to add `Deref<Target = Bound<T>> for BoundRef<T>` to make this work, but I see no problem with that impl.

Reply:
Nice, I don't see a problem with implementing `Deref` either, since `BoundRef` is really just `&Bound` 👍